OmniKit/Model/FaultEventCode.swift

```diff
@@ -117,197 +117,205 @@ public struct FaultEventCode: CustomStringConvertible {
+                return "Fault Event Type 0x8B: Corruption of $283, $2E3, $315 table"
             case .faultEventSetupPodType8D:
-                return LocalizedString("Fault Event Type 0x8D: Bad value problem #3 to verify_and_start_pump.", comment: "Fault Event Type 0x8D: Bad value problem #3 to verify_and_start_pump.")
+                return "Fault Event Type 0x8D: Bad value problem #3 to verify_and_start_pump"
             case .faultEventSetupPodType8E:
-                return LocalizedString("Fault Event Type 0x8E: Problem inside verify_and_start_pump.", comment: "Fault Event Type 0x8E: Problem inside verify_and_start_pump.")
+                return "Fault Event Type 0x8E: Problem inside verify_and_start_pump"
             case .faultEventSetupPodType8F:
-                return LocalizedString("Fault Event Type 0x8F: Bad value during verify_and_start_pump because of sub_ACE3 is stated as bad value.", comment: "Fault Event Type 0x8F: Bad value during verify_and_start_pump because of sub_ACE3 is stated as bad value.")
+                return "Fault Event Type 0x8F: Bad value during verify_and_start_pump because of sub_ACE3 is stated as bad value"
             case .badValueForTables:
-                return LocalizedString("Fault Event Type 0x90: Bad value for $283/$2E3/$315 table specification.", comment: "Fault Event Type 0x90: Bad value for $283/$2E3/$315 table specification.")
+                return "Fault Event Type 0x90: Bad value for $283/$2E3/$315 table specification"
             case .faultEventSetupPodType91:
-                return LocalizedString("Fault Event Type 0x91: Problem inside verify_and_start_pump.", comment: "Fault Event Type 0x91: Problem inside verify_and_start_pump.")
+                return "Problem inside verify_and_start_pump"
             case .faultEventSetupPodType92:
-                return LocalizedString("Fault Event Type 0x92: Problem inside verify_and_start_pump.", comment: "Fault Event Type 0x92: Problem inside verify_and_start_pump.")
+                return "Problem inside verify_and_start_pump"
             case .faultEventSetupPodType93:
-                return LocalizedString("Fault Event Type 0x93: Problem inside verify_and_start_pump.", comment: "Fault Event Type 0x93: Problem inside verify_and_start_pump.")
+                return "Problem inside verify_and_start_pump"
             case .badValueField6in0x1A:
-                return LocalizedString("Fault Event Type 0x95: Bad table specifier field6 in 1A command.", comment: "Fault Event Type 0x95: Bad table specifier field6 in 1A command.")
+                return "Bad table specifier field6 in 1A command"
             }
         } else {
-            return String(format: LocalizedString("Fault Event Code %1$02x", comment: "Format string for unknown Fault Event code."), rawValue)
+             return "Unknown Error Code"
         }
     }
 }
+
+extension String {
+    mutating func addFaultCodePrefix(rawValue: UInt8) {
```

Comment:
This is not being used.  Also, it can probably just be inline above, in the description var; no need to extend a system class here.